### DISCUSSION TITLE
能否添加向好友发送文件的api

### DISCUSSION THREAD

**Discussion by ericting** [Mrs4s/go-cqhttp#887] [Ideas]
是这样的，我们每次大学习都要截图上传到群文件（形式主义），
然后就有了搞一个bot来实现自动上传的想法，找了go-cqhttp文档并没有发现有向好友发送文件的api

**Comment by Mrs4s** [comment]
发送好友文件需要逆向一个native库以获取一个特殊的编码逻辑，难度还是比较高的，所以一直没做.
但是群文件有相关API，参考文档就行了.

**Reply by BobOfRivia** [reply]
我想尝试去解决这个问题，不知可否再提供一些已知的情报～

**Comment by BobOfRivia** [comment]
@ericting 
你的需求其实有个简单的备选方案，可以将文件上传到云端生成链接发到QQ端。

**Reply by zht163** [reply]
oss，请
